shell-ui/src/auth/permissionUtils.js

```diff
@@ -11,7 +11,7 @@ export const isEntryAccessibleByTheUser = (
   userGroups: string[],
 ): boolean => {
   return (
-    pathDescription.groups?.every((group) => userGroups.includes(group)) ??
+    pathDescription.groups?.some((group) => userGroups.includes(group)) ??
```

Comment:
Do we have mentions of this navbar features in some changelog? Could be useful for consumers (although for now, it wasn't yet released, so I think it's fine, just thinking about the future).

Reply:
done, I've added some missing items in the change-log

Reply:
Fine for now :) I was more thinking of creating a `shell-ui/CHANGELOG.md`, since it's not really user-facing, but we'll review all of this before releasing anyway.